openslides/agenda/static/js/agenda/site.js

```diff
@@ -546,6 +561,52 @@ angular.module('OpenSlidesApp.agenda.site', ['OpenSlidesApp.agenda'])
      }
 ])
 
+.controller('ListOfSpeakersViewCtrl', [
+    '$scope',
+    '$http',
+    'Projector',
+    'Assignment',
+    'Customslide',
+    'Motion',
+    'Agenda',
+    function($scope, $http, Projector, Assignment, Customslide, Motion, Agenda) {
+        $scope.$watch(
+            function() {
+                return Projector.lastModified(1);
+            },
+            function() {
+                Projector.find(1).then( function(projector) {
+                $scope.AgendaItem = null;
+                _.forEach(projector.elements, function(element) {
+                    if (element.name == 'motions/motion') {
```

Comment:
Suggestion: Use a switch case block. (This is only a suggestion. You can also open a new issue and we add the label clean up for later.)